docs/csv-import-tool/README.md

```diff
@@ -202,3 +202,29 @@ for row in csv.reader(stream, delimiter=',', doublequote=True,
 For more information about the meaning of the above values, see:
 https://docs.python.org/3/library/csv.html#csv.Dialect
 
+## Errors
+
+1. [Skipping duplicate node with ID '{}'. For more details, visit:
+   memgr.ph/csv-import-tool.](#error-1)
+2. [Skipping bad relationship with START_ID '{}'. For more details, visit:
+   memgr.ph/csv-import-tool.](#error-2)
+3. [Skipping bad relationship with END_ID '{}'. For more details, visit:
+   memgr.ph/csv-import-tool.](#error-3)
```

Comment:
The [memgr.ph/csv-import-tool](url) is not valid :)

Reply:
I will link it to this part of the documentation on the repo, but now I have a chicken and an egg problem :))
You can approve this, then I will create a short link to lead to this new part of the documentation and it will work then + I will update the code BCS in the real error it's just `memgr.ph/csv` not `memgr.ph/csv-import-tool`